tests/components/nws/snapshots/test_weather.ambr

```diff
@@ -197,12 +228,50 @@
     }),
   })
 # ---
+# name: test_forecast_service[hourly]
```

Comment:
Where did these come from? Does not see any other test changes that would create these.

Reply:
Good catch.  It might have been a result of the merge conflict merging and I messed it up? I will investigate it.

Reply:
Fixed in https://github.com/home-assistant/core/pull/117254/commits/5b4610a8792ec0fa59d8e45fc47f30b2208c47aa